web/react/components/channel_header.jsx

```diff
@@ -194,6 +194,17 @@ export default class ChannelHeader extends React.Component {
             );
         }
 
+        const popoverListMembers = [];
+        if (!isDirect) {
+            popoverListMembers.push(
+                <PopoverListMembers
+                    members={this.state.users}
+                    memberCount={this.state.userCount}
+                    channelId={channel.id}
+                />
```

Comment:
You might also need to make the first line `let popoverListMembers = null;` since React might throw an error if it is undefined. I'm not certain of that though.